resources/lang/en/events/lead_ceremony_df.json

```
@@ -1196,6 +1245,12 @@
       "star_trait": [],
       "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
       "life_giving": [
+        {
+          "text": "r_c approaches, casting a shrewd eye over m_c. Whatever this evaluation is looking for, r_c seems satisfied. {PRONOUN/r_c/subject/CAP} {VERB/r_c/promise/promises} m_c that this new life for [virtue] will ensure {PRONOUN/m_c/subject} can wreak all the vegeance {PRONOUN/m_c/subject} {VERB/m_c/want/wants}.",
```

```suggestion
          "text": "r_c approaches, casting a shrewd eye over m_c. Whatever this evaluation is looking for, r_c seems satisfied. {PRONOUN/r_c/subject/CAP} {VERB/r_c/promise/promises} m_c that this new life for [virtue] will ensure {PRONOUN/m_c/subject} can wreak all the vengeance {PRONOUN/m_c/subject} {VERB/m_c/want/wants}.",
```
Maybe instead of wreaking vengeance, something about how it will give them the added boost they need to ensure justice is always served. With that usual flavour of DF values we so cherish.